test/request-response-samples/RetrieveCustomerDetails__returning_a_date_should_parse_the_date_with_moment/wsdl_options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "momentDateTimeFormat": null,
```

Comment:
how about just `dateTimeFormat` with a note in the README that we're currently using moment for date time format in the wsdl options?